examples/mysql/README.md

```diff
@@ -0,0 +1,86 @@
+MySQL filter example
+===================
+
+In this example, we show how a [MySQL
+filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/network_filters/mysql_proxy_filter)
+can be used with the Envoy proxy. The Envoy proxy [configuration](./envoy.yaml)
+includes a MySQL filter that routes egress requests to a MySQL server instance
+running in a separate container.
```

Comment:
The MySQL filter is mainly used to parse queries and generate stats. It's not used for routing requests to a server. You should be able to do that without using the MySQL filter.

Reply:
Fixed.